.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,41 @@
+name: Bug report
+description: |
+  Report a bug. For security vulnerabilities see Report a security vulnerability in the templates.
+title: "BUG:"
+labels: [00 - Bug]
+
+body:
+- type: textarea
+  attributes: 
+    label: "Describe the issue:"
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: "Reproduce the code example:"
+    description: |
+      A short code example that reproduces the problem/missing feature. It should be self-contained, i.e., possible to run as-is via
+      'python myproblem.py'.
+    placeholder: |
+      ```python
+      import numpy as np
+      << your code here >>
+      ```
+    render: markdown
```

Comment:
```suggestion
    render: python
```
Surely?

Reply:
Yes that works, I'll change it.

Reply:
I'm curious; how are you testing this? I tried to set this up in another repo, and couldn't come up with anything beyond clobbering `main` on my fork.